### DISCUSSION TITLE
2021.06.14

### DISCUSSION THREAD

**Discussion by malaheaven** [malaheaven/issue-tracker#52] [General]
### 회의:
 1. login type처리 하는 부분 header의 User-Agent항목으로 변경
 User-Agent : 
 - fe: IssueTrackerFE
 - be: IssueTrackerIOS
 
 2. GET /api/users (유저 전체 리스트) 협의
 
3. FE callback url 변경

**Comment by choitree** [comment]
### [BE]ERD설계 

![image](https://user-images.githubusercontent.com/64007131/121859286-7b8f9b80-cd32-11eb-87d1-79ecdd59907c.png)

**Reply by choitree** [reply]
마일스톤과 개발해야되는 API의 이슈를 먼저 생성했습니다.
너무 많은 이슈가 생겼지만 놀라지말아주세요😀🥕

**Comment by Settpark** [comment]
iOS: 백엔드와 header관련 회의 했습니다. 회의 내용 적용하는 함수 작성했습니다. Network 관련 함수 작성하였습니다. iOS는 오늘 행복합니다.

**Comment by 17-sss** [comment]
**FE**
- 백앤드와 API 회의를 했습니다.
- 저번주까지의 진행상황을 정리 후 작업하였습니다.
- 현 프로젝트에서 다방면으로 쓰이는 Modal (이슈 필터, 마일스톤 리스트 등에 쓰이는 Modal) 생성했습니다
- 이전에는 써보지 못했던 다른 상태관리 라이브러리 (Recoil)에 대해 공부하였습니다.

**FE도 행복합니다 (●'◡'●)**
